Bottom: 27a5382ee02f047829e51cf1ebf914720c1677f4
Top:    19ccba4b82a0868671bb541fa5e65d08ae62bcec
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-14 12:35:07 +0800

Refresh of refresh-of-tuntap-introduce-tx

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 8eb60a3..501d40b 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -131,7 +131,7 @@ struct tap_filter {
 #define MAX_TAP_FLOWS  4096
 
 #define TUN_FLOW_EXPIRE (3 * HZ)
-#define TUN_RING_SIZE 256
+#define TUN_RING_SIZE 1000
 
 struct tun_pcpu_stats {
 	u64 rx_packets;
